stream_rails.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'actionpack', '>= 3.0.0'
   gem.add_dependency 'railties', '>= 3.0.0'
-  gem.add_dependency 'stream-ruby', '~> 2.5'
+  gem.add_dependency 'stream-ruby', '~> 2.5.9'
```

Comment:
Can you switch this back to the minor revision? Otherwise you're going to lock our Rails library to only patch revisions of stream-ruby 2.5.x going forward.